src/components/ChangePlan/ChangePlan.test.js

```diff
@@ -0,0 +1,442 @@
+import React from 'react';
+import { render, fireEvent, waitFor, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { act } from 'react-dom/test-utils';
+import IntlProvider from '../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import ChangePlan, {
+  BulletOptions,
+  OptionItem,
+  BasicBullet,
+  BigDataBullet,
+  NewLabel,
+  StarBullet,
+  FreeCard,
+  AgenciesCard,
+  CardWithPrice,
+} from './ChangePlan';
+import { MemoryRouter as Router } from 'react-router-dom';
+import { AppServicesProvider } from '../../services/pure-di';
+import { allPlans } from '../../services/doppler-legacy-client.doubles';
+
+describe('OptionItem component', () => {
+  it('should render OptionItem', async () => {
+    // Arrange
+    const text = 'fake test';
+
+    // Act
+    const { container, getByText } = render(
+      <OptionItem bullet={<BasicBullet />}>{text}</OptionItem>,
+    );
+
+    // Assert
+    expect(getByText(text)).toBeInTheDocument();
+    expect(container.querySelector('.dp-icodot')).toBeInTheDocument();
+  });
+});
+
+describe('BasicBullet component', () => {
+  it('should render BasicBullet', async () => {
+    // Act
+    const { container } = render(<BasicBullet />);
+
+    // Assert
+    expect(container.querySelector('.dp-icodot')).toBeInTheDocument();
+  });
+});
+
+describe('BigDataBullet component', () => {
+  it('should render BigDataBullet', async () => {
+    // Arrange
+    const text = 'fake test';
+
+    // Act
+    const { getByText } = render(<BigDataBullet>{text}</BigDataBullet>);
+
+    // Assert
+    expect(getByText(text)).toBeInTheDocument();
```

Comment:
Don`t we have a special bullet class for big query also?

Reply:
these are all who are:
- StarBullet
- BasicBullet
- BigDataBullet